rkt/session-df/native-series.rkt

```diff
@@ -1443,6 +1443,7 @@
          (define/override (plot-ticks) (time-ticks #:formats '("~H:~M")))
          (define/override (axis-label) "Time (hour:min)")
          (define/override (name) "Time")
+         (define/override (filter-width) 5.0) ; seconds
```

Comment:
The swimming plots are not currently filtered (the control is disabled).  Is there a reason why this was added here?

Reply:
Without this change I'm getting an exception:
```
2021-08-25 09:46:18.935 thread <worker-1>: #(struct:exn:fail:contract car: contract violation
  expected: pair?
  given: '#(#<path:../ActivityLog2/rkt/session-df/session-df.rkt> 1075 16 39264 34) #<continuation-mark-set>)
```
Here is corresponding place in code:
![exception_stelle](https://user-images.githubusercontent.com/4608274/130750290-92081f63-2cb0-4336-807c-ad210e485ecf.png)

`y-axis` is in this case  `axis-hr-bpm`

```
(define axis-hr-bpm
  (new (class series-metadata% (init) (super-new)
         (define/override (axis-label) "Heart Rate (bpm)")
         (define/override (should-filter?) #t)
         (define/override (series-name) "hr")
         (define/override (name) "HR")

         (define/override (factor-fn sport (sid #f))
           (define sz
             (if sid
                 (sport-zones-for-session sid 'heart-rate)
                 (sport-zones-for-sport (sport-id sport) (sub-sport-id sport) 'heart-rate)))
           (and sz
                (lambda (val)
                  (let ((zone (value->zone sz val)))
                    (zone->label zone)))))

         (define/override (factor-colors) (ct:zone-colors))


         )))
```
should I better create an override `axis-swim-hr-bpm` of `axis-hr-bpm` with `(define/override (should-filter?) #f)`?

Reply:
I think it is simpler to just have a filter-width, I just wasn't sure if this was actually needed -- looks like it is.